recipes/ufal.udpipe/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "ufal.udpipe" %}
+{% set version = "1.2.0.1" %}
+{% set sha256 = "e24ebb712232d657900aee2b2e25e7ab45726089796dee7a376faf22e619eeb7" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
+  skip: true  # [win and vc<14]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+  host:
+    - python
+    - pip
+  run:
+    - python
+
+test:
+  imports:
+    - ufal.udpipe
+
+about:
+  home: http://ufal.mff.cuni.cz/udpipe
+  license: MPL 2.0
+  license_family: OTHER
+  license_file: LICENSE
+  summary: 'UDPipe is a trainable pipeline for tokenization, tagging, lemmatization and dependency parsing of CoNLL-U files.'
+  description: |
+    UDPipe is a trainable pipeline for tokenization, tagging, lemmatization and 
+    dependency parsing of CoNLL-U files. UDPipe is language-agnostic and can be 
+    trained given annotated data in CoNLL-U format. Trained models are provided 
+    for nearly all UD treebanks. UDPipe is available as a binary for Linux/Windows/OS X, 
+    as a library for C++, Python, Perl, Java, C#, and as a web service. Third-party 
+    R CRAN package also exists.
+  doc_url: http://ufal.mff.cuni.cz/udpipe/users-manual
+  dev_url: https://github.com/ufal/udpipe
+
+extra:
+  recipe-maintainers:
+    - ajdapretnar
+    - primozgodec
+    - astaric
```

Comment:
All of these people already agreed to be a maintainer as well?

@primozgodec
@astaric

Reply:
I am ok with being a maintainer

Reply:
I agree